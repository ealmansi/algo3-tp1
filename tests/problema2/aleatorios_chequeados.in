5
3 6
15 13
7 8
12 17
13 6
5
1 19
18 18
18 11
11 20
6 7
5
17 11
14 12
9 13
7 12
11 7
5
12 1
13 13
13 19
15 11
8 16
5
6 20
17 1
11 3
14 7
13 1
5
12 20
3 10
2 17
8 3
9 8
5
20 10
15 12
10 4
6 19
14 17
5
16 14
6 19
13 4
10 16
17 18
5
9 3
12 20
16 11
5 5
4 15
5
16 5
17 16
19 2
20 18
14 17
5
10 5
15 1
5 15
2 7
5 14
0