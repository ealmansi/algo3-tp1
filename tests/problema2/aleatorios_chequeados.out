5 2 3 4 1 1230
3 2 5 4 1 1627
5 1 2 3 4 1575
1 4 2 3 5 1569
2 5 3 4 1 458
4 5 1 2 3 734
3 1 2 5 4 1684
3 1 5 4 2 1898
1 3 4 2 5 1030
3 1 4 2 5 2159
2 1 5 3 4 434